### ISSUE TITLE
 why can only reach 34 on Vimeo90K and 35.87 in papers

### ISSUE THREAD

**Issue by fanfanba** [soonwonh/FILM-pytorch#2]
why can only reach 34 on Vimeo90K and 35.87 in papers，Not enough iterations, or other problems?

**Comment by soonwonh** [comment]
Due to time constraint, I did not implemented various training details such as learning rate scheduling, hyper parameter optimization, etc. And also didn't experimented with enough iterations. May boost additional performance by optimizing and training better configuration.
